src/procman.cpp

```diff
@@ -347,8 +349,13 @@ procman_data_new (GSettings *settings)
     g_free (color);
 
     /* Sanity checks */
-    swidth = WidthOfScreen (gdk_x11_screen_get_xscreen (gdk_screen_get_default ()));
-    sheight = HeightOfScreen (gdk_x11_screen_get_xscreen (gdk_screen_get_default ()));
+    Gdk::Rectangle srect;
+    Gtk::Widget *app = Glib::wrap(pd->app);
+    Glib::RefPtr<Gdk::Window> window = app ? app->get_window() : Glib::RefPtr<Gdk::Window>();
```

Comment:
Note that `pd->app` is null initially, but `get_monitor_at_window()` is null-safe.

Reply:
New runtime warning:
```
gdk_window_get_origin: assertion 'GDK_IS_WINDOW (window)' failed
```
![Screenshot at 2020-08-30 18-28-17](https://user-images.githubusercontent.com/10171411/91664510-fcf1a680-eaef-11ea-8487-6d0c18308a40.png)

pd->app is null 0x0

![Screenshot at 2020-08-30 18-57-45](https://user-images.githubusercontent.com/10171411/91665018-147e5e80-eaf3-11ea-9325-d7ebd60cc763.png)

Reply:
Perhaps you need to check the height and width of the window (CLAMP) once it has been created in create_main_window.